CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Introduce new dynamic cluster setting to control slice computation for concurrent segment search ([#9107](https://github.com/opensearch-project/OpenSearch/pull/9107))
 - Implement on behalf of token passing for extensions ([#8679](https://github.com/opensearch-project/OpenSearch/pull/8679))
 - Provide service accounts tokens to extensions ([#9618](https://github.com/opensearch-project/OpenSearch/pull/9618))
+- Configurable merge policy for index with an option to choose from LogByteSize and Tiered merge policy ([#9992](https://github.com/opensearch-project/OpenSearch/pull/9992))
+- Add search query categorizor ([#10255](https://github.com/opensearch-project/OpenSearch/pull/10255))
```

Comment:
Oh -- we're likely to release this in 2.x, so please move the changelog entry down to the `Unreleased 2.x` section below.

Also, the "configurable merge policy" change did go out in 2.11, so it doesn't need to appear in any of the "Unreleased" sections.